Changelog.md

```diff
@@ -1,15 +1,9 @@
-### 0.8.5 (unreleased)
-
-Language Features:
```

Comment:
Huh, why were these removed?

Reply:
I assumed we were taking breaking off 0.8.4, but that is obviously incorrect. @christianparpart could you please fix this?